linkerd/app/core/src/control.rs

```diff
@@ -33,6 +33,13 @@ impl fmt::Display for ControlAddr {
     }
 }
 
+// Needed for metrics labels, admittedly kind of weird...hm.
+impl From<&'_ ControlAddr> for ControlAddr {
+    fn from(addr: &Self) -> Self {
+        addr.clone()
+    }
+}
+
```

Comment:
Is this comment correct? It seems to compile if I remove this impl..

Reply:
oh, yup --- this is an artifact of an earlier implementation. will remove it, thanks for noticing!